### PR TITLE
Make intro less Fedora specific

### PR DESCRIPTION
**Pull request by ncoghlan**

### CODE REVIEW

**Comment by ncoghlan** [comment]
I mentioned to @ralphbean at PyCon that the fedmsg docs still gave the impression that it was a Fedora Infrastructure specific project, and finally found the time to put together a concrete suggestion for ways to reduce that impression :)

**Comment by ncoghlan** [comment]
The CI run actually passed, the failure is due to #376 causing things to fail on Python 2.6

**Comment by pypingou** [comment]
I like it! :)

**Comment by sayanchowdhury** [comment]
:+1: for me

**Comment by pypingou** [comment]
Let's merge :)

File: doc/index.rst [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+after it was also adopted for
+`use in Debian's infrastructure <https://wiki.debian.org/FedMsg>`_.
+
 `Click here to see a feed of the Fedora bus
 <https://apps.fedoraproject.org/datagrepper/raw>`_.  There's also a
 ``#fedora-fedmsg`` channel on the freenode network with a firehose bot echoing
 messages to channel to help give you a feel for what's going on.
 
-You can find the list of available topics in our infrastructure
+You can find the list of available topics in Fedora' s infrastructure
```
Comment (glensc) [inline doc/index.rst]: extra space

File: doc/index.rst [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
-one another over a message bus instead of communicating with each other in
-the heterogenous, "Rube-Goldberg" ways they do now.
+Some time ago, the Fedora Infrastructure team wanted to hook all the services in
+Fedora Infrastructure up to send messages to one another over a message bus instead
+of communicating with each other in the heterogenous, "Rube-Goldberg" way they did
+previously.
 
-``fedmsg`` (Fedora-Messaging) is a python package and API used around
-Fedora Infrastructure to send and receive messages to and from applications.
+``fedmsg`` (FEDerated MeSsaGe bus) is a python package and API defining a brokerless
```
Comment (glensc) [inline doc/index.rst]: `python` should be probably written as `Python`

**Pull request merged**